devfile.yaml

```diff
@@ -18,4 +18,4 @@ commands:
       - type: exec
         component: maven
         command: mvn clean install -DskipTests
-        workdir: ${CHE_PROJECTS_ROOT}/che
+        workdir: ${CHE_PROJECTS_ROOT}/che-server
```

Comment:
can we make non-subject-related changes in separate pr?

Reply:
I  have removed this quick fix.